### PR TITLE
160184959 Include detected changes in pre notice email

### PR DESCRIPTION
**Pull request by tatut**
# Added
* Pre notice email includes changes detected from GTFS packages

### CODE REVIEW

File: ote/test/clj/ote/tasks/pre_notices_test.clj [hunk 1]
```diff
@@ -0,0 +1,86 @@
+(ns ote.tasks.pre-notices-test
+  (:require [ote.tasks.pre-notices :as sut]
```
Comment (rckopteri) [inline ote/test/clj/ote/tasks/pre_notices_test.clj]: ":as pre-notices" would be easier to understand

File: ote/test/clj/ote/tasks/pre_notices_test.clj [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+  (testing "Nothing is sent if the regions don't match"
+    (http-post "admin" "settings/email-notifications"
+               {::user-notifications/finnish-regions ["02"]})
+    (send!)
+    (is (empty? @outbox)))
+
+  (testing "Email is sent when regions match"
+    (http-post "admin" "settings/email-notifications"
+               {::user-notifications/finnish-regions ["01" "02"]})
```
Comment (rckopteri) [inline ote/test/clj/ote/tasks/pre_notices_test.clj]: Does sql queries work if finnish-regions are in wrong order? e.g.  ["02" "01"] ?

File: ote/test/clj/ote/tasks/pre_notices_test.clj (hunk 1 above)
Comment (tatut) [reply ote/test/clj/ote/tasks/pre_notices_test.clj]: true, the "sut" name comes by default when creating test namespaces in CIDER and refers to System Under Test

File: ote/test/clj/ote/tasks/pre_notices_test.clj (hunk 2 above)
Comment (tatut) [reply ote/test/clj/ote/tasks/pre_notices_test.clj]: Yes, due to `&&` (overlap) operator used in queries

**Pull request merged**